components/Votes/ActiveVotes.tsx

```diff
@@ -311,33 +309,29 @@ export function ActiveVotes() {
     selectVote(undefined, vote);
   }
 
+  const data = entriesToShow?.map((vote, index) => ({
```

Comment:
now instead of passing in data rows as react elements, we just pass in the data and do the iteration in the `VoteList` component instead